src/OmniSharp/app.config

```diff
@@ -3,4 +3,20 @@
   <startup>
     <supportedRuntime version="v4.0" sku=".NETFramework,Version=v4.6" />
   </startup>
+  <runtime>
```

Comment:
this was added so that a refactoring written against Roslyn 1.0.0 (which is often the case) can load too